### PR TITLE
Add build for IBMz

### LINKED ISSUE #2160: [Bug]:  Jaeger UI does not build on IBM s390x arch (big endian)
### What happened?

 Jaeger UI does not build on IBM s390x arch (big endian) 
 
 The Vite 5 uses rollup 4 which does not work on s390x. The workaround with wasm-node fails as well. 

see:
* https://github.com/rollup/rollup/issues/5354 
* https://github.com/rollup/rollup/pull/5346
* https://github.com/rollup/rollup/pull/5350

### Steps to reproduce

https://github.com/jaegertracing/jaeger-ui/pull/2119

### Expected behavior

Be able to build Jaeger UI natively on s390x. It is Red Hat internal requirement that everything has to be built directly from source code. 

### Relevant log output

```shell
#9 568.4 $ NODE_ENV=production REACT_APP_VSN_STATE=$(../../scripts/get-tracking-version.js) vite build
#9 576.0 vite v5.0.11 building for production...
#9 576.5 ✓ 0 modules transformed.
#9 576.5 [vite:build-html] memory access out of bounds
#9 576.5 file: /workspace/packages/jaeger-ui/index.html
#9 576.5 error during build:
#9 576.5 RuntimeError: memory access out of bounds
#9 576.5     at wasm://wasm/0063bf96:wasm-function[1558]:0x162080
#9 576.5     at wasm://wasm/0063bf96:wasm-function[249]:0x104dc0
#9 576.5     at wasm://wasm/0063bf96:wasm-function[1442]:0x15f2f9
#9 576.5     at module.exports.xxhashBase64Url (/workspace/node_modules/rollup/dist/wasm-node/bindings_wasm.js:159:14)
#9 576.5     at getXxhash (file:///workspace/node_modules/rollup/dist/es/shared/node-entry.js:16781:12)
#9 576.5     at FileEmitter.assignReferenceId (file:///workspace/node_modules/rollup/dist/es/shared/node-entry.js:18266:27)
#9 576.5     at FileEmitter.emitAsset (file:///workspace/node_modules/rollup/dist/es/shared/node-entry.js:18311:34)
#9 576.5     at FileEmitter.emitFile (file:///workspace/node_modules/rollup/dist/es/shared/node-entry.js:18181:25)
#9 576.5     at Object.emitFile (file:///workspace/node_modules/rollup/dist/es/shared/node-entry.js:17585:41)
#9 576.5     at fileToBuiltUrl (file:///workspace/node_modules/vite/dist/node/chunks/dep-V3BH7oO1.js:16718:43)
#9 576.7 error Command failed with exit code 1.
#9 576.7 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
#9 ERROR: process "/bin/sh -c yarn install --frozen-lockfile && cd ./packages/jaeger-ui && yarn build" did not complete successfully: exit code: 1
------
 > [4/4] RUN  yarn install --frozen-lockfile && cd ./packages/jaeger-ui && yarn build:
576.5     at wasm://wasm/0063bf96:wasm-function[1442]:0x15f2f9
576.5     at module.exports.xxhashBase64Url (/workspace/node_modules/rollup/dist/wasm-node/bindings_wasm.js:159:14)
576.5     at getXxhash (file:///workspace/node_modules/rollup/dist/es/shared/node-entry.js:16781:12)
576.5     at FileEmitter.assignReferenceId (file:///workspace/node_modules/rollup/dist/es/shared/node-entry.js:18266:27)
576.5     at FileEmitter.emitAsset (file:///workspace/node_modules/rollup/dist/es/shared/node-entry.js:18311:34)
576.5     at FileEmitter.emitFile (file:///workspace/node_modules/rollup/dist/es/shared/node-entry.js:18181:25)
576.5     at Object.emitFile (file:///workspace/node_modules/rollup/dist/es/shared/node-entry.js:17585:41)
576.5     at fileToBuiltUrl (file:///workspace/node_modules/vite/dist/node/chunks/dep-V3BH7oO1.js:16718:43)
576.7 error Command failed with exit code 1.
576.7 info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
------
WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
Dockerfile:7
--------------------
   5 |     COPY . .
   6 |     
   7 | >>> RUN  yarn install --frozen-lockfile && cd ./packages/jaeger-ui && yarn build
   8 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c yarn install --frozen-lockfile && cd ./packages/jaeger-ui && yarn build" did not complete successfully: exit code: 1
```
```


### Screenshot

_No response_

### Additional context

_No response_

### Jaeger backend version

main

### SDK

_No response_

### Pipeline

_No response_

### Stogage backend

_No response_

### Operating system

_No response_

### Deployment model

_No response_

### Deployment configs

_No response_

### PR DESCRIPTION
**Pull request by pavolloffay**
Red Hat offers Jaeger as part of the OpenShift layered product. We have internal requirement to build UI from source for all supported platforms. The UI was not building on IBM P/Z, there was a fix in the rollup 4 (used by vite 5) and we would like to check if the UI builds per PR.

Resolves #2160

1. add resolutions to the package.json
2. run `yarn install`

### CODE REVIEW

**Comment by pavolloffay** [comment]
Example build run: https://github.com/jaegertracing/jaeger-ui/actions/runs/8736907060/job/23972742438 (~17 min)

File: package.json [hunk 1]
```diff
@@ -63,5 +64,8 @@
     "hooks": {
       "pre-commit": "npm-run-all -ln --parallel lint test"
     }
+  },
+  "dependencies": {
+    "rollup": "4.13.1"
```
Comment (yurishkuro) [inline package.json:69]: can you elaborate what this dependency is and why it needs to be a runtime dep?

Comment (pavolloffay) [reply package.json:69]: good catch, this should be dev dependency.

The jaeger-ui uses Vite5 that uses rolloup to build the UI. The vite5 uses by default rollup 4.13.0, however the 4.13
.1 contains the fix for the IBMz build.

Comment (yurishkuro) [reply package.json:69]: then you want the resolution to be `"^4.13.1"`, not the exact pin.

Comment (pavolloffay) [reply package.json]: done,

**Comment by pavolloffay** [comment]
@yurishkuro PR is updated. 

Any idea why the lint failed?

**Comment by yurishkuro** [comment]
I think because you bumped a lot of other dependencies in the lock file, it should be limited to just this one package.

**Comment by pavolloffay** [comment]
Done the build now passes

**Review by yurishkuro — APPROVED** [review]

**Pull request merged**